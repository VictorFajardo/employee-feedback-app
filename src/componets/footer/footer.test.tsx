import { shallow } from 'enzyme';
import Footer from './footer';

it('expect render Footer component', () => {
  expect(shallow(<Footer />)).toMatchSnapshot();
});
