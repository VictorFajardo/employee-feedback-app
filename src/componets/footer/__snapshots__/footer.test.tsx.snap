// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render Footer component 1`] = `ShallowWrapper {}`;
